src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is the most commonly used approach. This may not be the right approach if a strong application service continuity during the migration is required. However, it may be a viable method if downtime is permitted. It works particularly well when the application accurately maintains [primary key](primary-key.html) and [foreign key](foreign-key.html) constraints.
+
+## Pros and Cons
+
+On the spectrum of different data migration strategies, 'lift and shift' has the following pros and cons. The terms "low" and "high" are not absolute, but relative to other approaches.
+
+Pros:
+
+- Low effort: in general, if you can afford a bit of downtime, the overall effort will be lower
+- Low cost: in general, "lift and shift" costs less than having your development team modify an application to write to two data sources, or run some kind of syncing between source and target systems
+- Low time start-to-finish: in general, the more downtime you can afford, the shorter the overall migration timeframe can be
+- Low technical risk: it does not involve running multiple systems alongside each other for an extended period of time
+- Good import/export tooling is available
+- Easy to practice dry runs of import/export using testing/non-production systems
+- Conceptually straightforward
+
+Cons:
+
+- High project risk: it involves at least some downtime and it either works or does not work
+- Application service continuity requirements must be relaxed (i.e., some kind of application downtime or degradation is usually needed)
+- Less control of the pace (*has* to be completed to meet a given downtime / service degradation window)
+
+## Process design considerations
+
+{{site.data.alerts.callout_info}}
+The high-level considerations in this section only refer to the data loading portion of your migration. They assume you are following the steps in the overall migration process described in [Migrate your Database to CockroachDB](migration-overview.html).
+{{site.data.alerts.end}}
+
+Keep in mind the following considerations when designing a 'Lift and Shift' data migration process.
+
+- [Decide how much downtime you can tolerate](#downtime)
+- [Decide if you will use a managed migration service](#managed-migration)
```

Comment:
Should this be retitled to, Decide your data migration tooling? (it seems more immediately obvious what to decide on)

Reply:
Sounds good - I have updated to use that section title, and added some text in that section directing users to see the other sections for more info about `IMPORT INTO` etc. if they are not using a managed service